### PR TITLE
Analyses carousel

### LINKED ISSUE #7880: Analysis workflow view is not scrollable, so hides some analyses
screen shot 2016-06-16 at 12 06 52

A3 and A2 in the screenshot, so they can't be accessed from this view.

### PR DESCRIPTION
**Pull request by nobuti**
This PR fixes #7880.

analyses-scroll

cc @viddo @xavijam @piensaenpixel

### CODE REVIEW

**Comment by xavijam** [comment]
Could you take a look at the stylesheets?

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analyses-workflow-list-view.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  },
+
+  _renderWorkflowItem: function (formModel) {
+    var view = new AnalysesWorkflowItemView({
+      analysisNode: this.analysis.findNodeById(formModel.id),
+      formModel: formModel,
+      viewModel: this.model,
+      layerId: this._layerId
+    });
+    this.$('.js-list').append(view.render().el);
```
Comment (viddo) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analyses-workflow-list-view.js]: Looking at the template and this view it looks like the <ul class="HorizontalBlockList"> is duplicated, right? I think the <ul> can be removed from the template and just use `this.$el.append(…)` here.

**Comment by viddo** [comment]
👍

**Comment by piensaenpixel** [comment]
fixing hound's alerts, for me ok, (take a look please the !important alert)

**Pull request merged**